src/run.js

```diff
@@ -9,26 +9,31 @@ module.exports = ({ testPath, config, globalConfig }) => {
   const start = new Date();
   const contents = fs.readFileSync(testPath, "utf8");
 
-  const isPretty = prettier.check(contents, { filepath: testPath });
+  return prettier.resolveConfig("./").then(config => {
```

Comment:
I think it would be better to pass the file path being checked rather than the current directory. It looks to me like the Prettier API prefers this for more granular configurations https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath-options.

Reply:
Good point, completely overlooked this. Thanks!